### DISCUSSION TITLE
How to disable pagination in the table widget ?

### DISCUSSION THREAD

**Discussion by ghost** [filamentphp/filament#1890] [Help] [answered]
How can I either:
- disable pagination
- or show previous/next without the pages selections

<img width="539" alt="Capture d’écran 2022-03-12 à 13 45 10" src="https://user-images.githubusercontent.com/5849907/158018477-67c4848d-7473-4d71-bc11-dab3f4e37d07.png">

**Comment by intrepidws** [comment, marked as answer]
Try this out:

https://filamentphp.com/docs/2.x/tables/getting-started#simple-pagination

**Reply by danharrin** [reply]
We should make this the default for widgets

**Comment by ryangjchandler** [comment]
PR for this here: #1908.
